x-pack/plugins/security_solution/server/lib/detection_engine/scripts/create_threat_data.sh

```diff
@@ -22,7 +22,7 @@ do {
 curl -s -k \
   -H "Content-Type: application/json" \
   -u ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD} \
-  -X PUT ${ELASTICSEARCH_URL}/mock-threat-list/_doc/$i \
+  -X PUT ${ELASTICSEARCH_URL}/mock-threat-list-1/_doc/$i \
```

Comment:
Is this change because `mock-threat-list` pushes the boundaries too much?

Reply:
Yeah, the original we have, so I swapped it for a smaller one but kept the larger one for testing.